pygdf/_gdf.py

```diff
@@ -132,6 +134,21 @@ def np_to_gdf_dtype(dtype):
     }[np.dtype(dtype).type]
 
 
+def gdf_to_np_dtype(dtype):
+    """Util to convert gdf dtype to numpy dtype.
+    """
+    return np.dtype({
+         libgdf.GDF_FLOAT64: np.float64,
+         libgdf.GDF_FLOAT32: np.float32,
+         libgdf.GDF_INT64: np.int64,
+         libgdf.GDF_INT32: np.int32,
+         libgdf.GDF_INT16: np.int16,
+         libgdf.GDF_INT8: np.int8,
+         libgdf.GDF_DATE64: np.dtype('datetime64[ms]'),
+         libgdf.N_GDF_TYPES: np.int32,
+     }[dtype])
```

Comment:
This dict should be defined at module-level scope, rather than being reinstantiated each call.